src/models/GitHub.js

```diff
@@ -61,8 +59,8 @@ class GitHub extends Fetcher {
   }
 
   // https://developer.github.com/v3/search/#search-issues
-  getTasks(query = Config.searchQuery) {
-    const params = `?per_page=${this.perPage}&q=${encodeURIComponent(query)}`
```

Comment:
Why did pagination go away? Just to get tests passing and land this branch since it should be easy to add back later?